backend/workhub/app.js

```diff
@@ -0,0 +1,43 @@
+var createError = require('http-errors');
+var express = require('express');
```

Comment:
We can use `const`s in here.

`const` is constants variables
`let` is normal variable with block scope in es6